sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -200,7 +215,9 @@ async def load(*args, **kwargs) -> "AzureAppConfigurationProvider":
         raise ValueError("A keyvault credential and secret resolver can't both be configured.")
 
     headers = _get_headers("Startup", **kwargs)
-    provider = _buildprovider(connection_string, endpoint, credential, **kwargs)
+    provider = _buildprovider(
+        connection_string, endpoint, credential, uses_key_vault="UsesKeyVault" in headers, **kwargs
```

Comment:
Is this change related to Feature Flags or was it just included?

Reply:
I think this was a bug in async where key vault usage was missed.